CodeTheDream/ride-share-app
# changed login prompt from username to email

## Description

What does the PR do?
--> Instead of "username", the login prompt now asks for "email".

## Background Information
It was not straightforward to ask for a username when it really is an email address.

## Screenshots
![Screen Shot 2021-01-25 at 16 02 13](https://user-images.githubusercontent.com/63831504/105771770-b2514a80-5f26-11eb-8463-93725ecaa4f2.png)

## Checklist:
- [v] Screenshots attached (if dealing with UI)

## Testing Steps for reviewer:
- Step 1
click on the login button
- Step 2
see the Sign in page with the placeholder title "email".
